streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/ProjectController.java

```diff
@@ -77,6 +82,16 @@ public RestResponse get(Project project) {
   @RequiresPermissions("project:build")
   @PermissionScope(team = "#project.teamId")
   public RestResponse build(Project project) throws Exception {
+    Long currentBuildCount = projectService.selectCurrentBuildCount();
```

Comment:
Don’t write logic code at the Controller layer, You can move the code in to the build method of ProjectServiceImpl, 

```
 ApiAlertException.throwIfTrue(
          currentBuildCount > maxProjectBuildNum,
          String.format(
              "The number of running Build projects exceeds the maximum number: %d of project max build",
              maxProjectBuildNum)
      );

```